fix: Change order consignment outlet position from REPLACE to AFTER

JedidiahAwuku opened the pull request:
Add status container to meet UI requirements

Remove duplicate code in pickup instore order consignment component file

Remove duplicate code in pickup-in-store.i18n file

Remove unused style from pickup instore order consignment scss file

JedidiahAwuku committed:
fix: Change order consignment outlet position from replace to after

Add status container to meet UI requirements

Remove duplicate code in pickup instore order consignment component file

Remove duplicate code in pickup-in-store.i18n file

Remove unused style from pickup instore order consignment scss file

WeizhengSap commented on feature-libs/pickup-in-store/root/components/container/pickup-in-store-order-consignment/pickup-in-store-order-consignment-container.module.ts:
```diff
@@ -20,7 +20,7 @@ import { PickupInStoreOrderConsignmentContainerComponent } from './pickup-in-sto
   providers: [
     provideOutlet({
       id: OrderOutlets.ORDER_CONSIGNMENT,
```
need clean the imports

JedidiahAwuku replied:
Done. Thank you

JedidiahAwuku committed:
fix: address code review

Remove list status date container from order consigned entries component

Remove 'order' from pick up order consignment container
as no longer needed

JedidiahAwuku committed:
Merge branch 'develop' into fix/CXSPA-2928

JedidiahAwuku committed:
refactor: pickup in store consignment component

Remove unwanted imports from module file

clean up code in component and html files

WeizhengSap commented on feature-libs/pickup-in-store/root/components/container/pickup-in-store-order-consignment/pickup-in-store-order-consignment-container.component.ts:
```diff
@@ -19,37 +20,19 @@ export type IOutletContextData = { item: Consignment; order: Order };
 
-    return addresses?.filter(Boolean).join(', ');
-  }
+  pointOfService$: Observable<PointOfService>;
```
maybe make is as `@Input()`. So, it can be used as child component as well.

WeizhengSap replied:
It can be `undefined`, if there is no outlet.

JedidiahAwuku replied:
Added. Thanks

WeizhengSap commented on feature-libs/pickup-in-store/root/components/container/pickup-in-store-order-consignment/pickup-in-store-order-consignment-container.component.ts:
```diff
@@ -4,11 +4,12 @@
+import { Observable } from 'rxjs';
+import { filter, map } from 'rxjs/operators';
 
 export type IOutletContextData = { item: Consignment; order: Order };
```
order is not used

JedidiahAwuku replied:
Sure. Removed

JedidiahAwuku committed:
fix: Address changes requested from code review

Remove order as it is not being used

Update doc file for PickupInStoreOrderConsignmentContainerComponent

Fix font weight for pickup table

JedidiahAwuku committed:
fix: Address changed requested in code review

JedidiahAwuku committed:
fix: remove order from cxOutletContext

WeizhengSap commented on feature-libs/pickup-in-store/root/components/container/pickup-in-store-order-consignment/pickup-in-store-order-consignment-container.component.ts:
```diff
@@ -4,52 +4,35 @@
+    this.pointOfService$ = this.outlet?.context$?.pipe(
+      map((context) => context.item?.deliveryPointOfService),
+      filter(
+        (pointOfService): pointOfService is PointOfService => !!pointOfService
```
maybe this `filter(isNotUndefined)`

JedidiahAwuku committed:
Merge branch 'develop' into fix/CXSPA-2928
